src/swap/components/SwapButton.tsx

```diff
@@ -6,7 +6,7 @@ import { buildSwapTransaction } from '../core/buildSwapTransaction';
 import { cn } from '../../utils/cn';
 import { isSwapError } from '../core/isSwapError';
 
-export function SwapButton({ onSubmit }: SwapButtonReact) {
+export function SwapButton({ disabled, onSubmit }: SwapButtonReact) {
```

Comment:
Should this have the default `false`? I understand it works as `undefined`, but doesn't seem correct.